src/services/auth/auth.test.ts

```diff
@@ -19,14 +17,16 @@ describe('Auth:', () => {
         const userPayload = { name: user.name, email: user.email, password: user.password };
         await createUser(userPayload);
 
+        const email = user.email || '';
+        const password = user.password || '';
```

Comment:
```suggestion
        const userPayload = { name: user.name, email: user.email, password: 'password' };
        await createUser(userPayload);

        const email = user.email;
        const password = 'password';
```

Better to do the `|| ''` to bypass typescript, I think 🙈